content/en/building/guides/interoperability/openmrs.md

```diff
@@ -0,0 +1,373 @@
+---
+title: "OpenMRS Interoperability"
+linkTitle: "OpenMRS"
+weight: 4
+description: >
+   Exchange data with systems based on OpenMRS using FHIR APIs
+keywords: openmrs interoperability
+relatedContent: >
+  building/guides/interoperability/cht-config
+  building/concepts/interoperability
+  building/guides/interoperability/openhim
+---
+
+[OpenMRS](https://openmrs.org) is an open-source electronic medical record system used in dozens of countries. Integrating CHT apps with OpenMRS can open up opportunities to improve health outcomes for patients by promoting better coordination of care. For example, referrals by CHWs in the community can be sent electronically to health facilities using OpenMRS so that nurses and clinicians can prepare for their visit and have full access to the assessment done by a CHW.
+
+Integrating CHT apps with OpenMRS can be achieved using the CHT's [Interoperability Tools]({{< ref "building/guides/interoperability/openhim" >}}) and following the guidance in the [Building interoperability with FHIR APIs documentation]({{< ref "building/guides/interoperability/cht-config" >}}). 
+
+## Overview
+
+The CHT's interoperability tools support interoperability with OpenMRS in a variety of ways:
+
+1. Sending patient and patient contact data.
+1. Sending reports data (encounters and observations).
+1. Receiving reports data (encounters and observations) from OpenMRS.
+
+The steps to create an OpenMRS interoperability workflow are:
+
+1. Profile the workflow in terms of what data needs to be exchanged between OpenMRS and the CHT application.
+1. Set up a test environment including an OpenMRS instance, a CHT instance, and OpenHIM. The [interoperability project](https://github.com/medic/cht-interoperability) has docker compose files so that you can set this up locally by running `startup.sh up-openmrs` in the interoperability project.
```

Comment:
```suggestion
1. Set up a test environment including an OpenMRS instance, a CHT instance, and OpenHIM. The [interoperability project](https://github.com/medic/cht-interoperability) has docker compose files so that you can set this up locally by running `./startup.sh up-openmrs` in the interoperability project.
```